src/components/presentations/Profilepage/Profilepage.jsx

```diff
@@ -34,16 +38,32 @@ class Profilepage extends Component {
     this.setState({ currentTab: tab });
   };
 
+  handleChange = async () => {
+    const form = new FormData();
+    const imageData = document.querySelector('input[type="file"]').files[0];
+    form.append('file', imageData);
+    const res = await uploadToCloudnary(form);
+    this.setState({ profilePic: res.url });
+  };
+
   render() {
-    const { currentTab, firstname, lastname } = this.state;
+    const { currentTab, firstname, lastname, profilePic } = this.state;
     return (
       <React.Fragment>
         <div className="profile-header">
           <div className="profile-img-container">
-            <img
-              src="https://res.cloudinary.com/dcn7hu7wo/image/upload/v1556979121/IMG_1763.jpg"
-              alt="avatar"
-            />
+            <img src={profilePic} alt="avatar" />
+            <form className="custom-file-upload" encType="multipart/form-data">
+              <label htmlFor="file-upload">
+                <i className="fa fa-upload" />
+                <input
+                  type="file"
+                  id="file-upload"
+                  name="file"
+                  onChange={this.handleChange.bind(this)}
```

Comment:
@sojida nice job with this implementation. can we use an arrow function for this binding instead... something along this lines `onChange={() => this.handleChange}`. would make this look a-lot cleaner 🙇🏽‍♂️